Server/Main.cs

```diff
@@ -87,6 +87,8 @@ public static TimeSpan ProfileTime
 		}
 
 		public static bool Service { get; private set; }
+
+        public static bool NoConsole { get; private set; }

```

Comment:
The goal of this PR is not to remove all user interaction. Core.Service and -service do this effectively. The primary goal of this PR is to allow the administrator to force the use of IO methods which do not require a console. 
This frees the administrator to attach to the IO streams of the ServUO application and supply input and read output with third party applications.